spec/index.html

```diff
@@ -734,9 +735,153 @@ <h3>Overview</h3>
     <section id="hash-1d-quads-exampkes" class="informative">
       <h3>Examples</h3>
 
-      <p class="ednote">
-        Add some examples where the algorithm results are unique or not.
-      </p>
+      <aside id="ex-unique-hashes"
+            class="example"
+            title="Unique hashes">
+        <p>This example illustrates hashing quads containing
+          blank nodes where hashing the statements mentioning
+          those blank nodes generates unique results.</p>
+
+        <figure id="fig-unique-hashes" style="text-align:center">
+          <!-- Source for this file is at https://docs.google.com/drawings/d/1LQ6fp4a35lrEOKRMD20gcncUkoitX0eW8QI_U2mQnAs/edit?usp=sharing -->
+          <object data="unique-hashes.svg" style="width: 75%" type="image/svg+xml" aria-describedby="fig-unique-hashes-alt">
+            <p id="fig-unique-hashes-alt">
+              The image represents the graph described in
+              <a href="#ex-unique-hashes-input">
+                the following code block
+              </a>.</p>
+          </object>
+          <figcaption>An illustration of a graph resulting in unique hashes.<br/>
+            Image available in
+            <a href="unique-hashes.svg" title="SVG image of a unique hashes">
+              <abbr title="Scalable Vector Graphics">SVG</abbr>
+            </a>.</figcaption>
+        </figure>
+
+        <pre id="ex-unique-hashes-input"
+             data-transform="updateExample"
+             title="Graph with blank nodes resulting in unique hashes">
+          <!--
+            :p :q _:b0 .
+            :p :r _:b1 .
+            _:b0 :p _:b2 .
+            _:b1 :p _:b2 .
+          -->
+        </pre>
+
+        <p>The algorithm will be called twice, with each blank node (`_:b0` and `_:b1`).
+          Running the algorithm with the reference node `_:b0` results in the
+          following quads, after replacing blank nodes:</p>
+
+        <pre data-transform="updateExample">
+          <!--
+            :p :q _:a .
+            _:a :p _:z .
+          -->
+        </pre>
+
+        <p>These are then serialized to N-Quads, concatenated and hashed resulting in
+          `3b26142829b8887d011d779079a243bd61ab53c3990d550320a17b59ade6ba36`.</p>
+
+        <p>The algorithm is run a second time with the reference node `_:b1` resulting
+          in the following quads:</p>
+
+        <pre data-transform="updateExample">
+          <!--
+            :p :r _:a .
+            _:a :p _:z .
+          -->
+        </pre>
+
+        <p>These are then serialized to N-Quads, concatenated and hashed resulting in
+          `3b26142829b8887d011d779079a243bd61ab53c3990d550320a17b59ade6ba36`.</p>
+
+        <p>Thus the generated hashes each reference just a single blank node,
```

Comment:
See above -- the hashes reference multiple blank nodes -- but only because of `_:b2` which could become a non-blank node to remedy that.

Reply:
You're correct; the example evolved from the original I had. But `_:b0 :p _:b2` and `_:b1 :p _:b2` would exhibit the more complicated scenario of requiring an N-Degree hash, which I was introducing in the next example. Better to replace the graph with something not requiring an N-Degree hash.

Reply:
Following your suggestion, I'll re-work this example to the following:

```
:p :q _:b0 .
:p :r _:b1 .
_:b0 :p :s .
_:b1 :r :s .
```

This should meet the original intention of this example, to have two blank nodes which can be uniquely hashed and not require the use of the N-Degree hash algorithm.